src/Core/Grid/Query/CategoryQueryBuilder.php

```diff
@@ -158,6 +159,13 @@ private function getQueryBuilder(array $filters)
                 'c.id_category = cs.id_category AND cs.id_shop = c.id_shop_default'
         );
 
+        $qb->leftJoin(
```

Comment:
If you group by on something present in the left join, it should be an Inner join :thinking:

Reply:
Nice catch ;)